gulpfile.js

```diff
@@ -81,6 +83,22 @@ var DEFINES = {
   PDFJS_NEXT: false,
 };
 
+function safeSpawnSync(command, parameters, options) {
+  // Windows has problems executing commands outside a shell.
+  if (process.platform === 'win32') {
+    options = options || {};
+    options.shell = true;
+  }
+
+  var result = spawnSync(command, parameters, options);
+  if (result.status) {
```

Comment:
`spawnSync` may exit with result status `null`, which is a valid value according to https://github.com/nodejs/node/pull/11288/files (see the IRC discussion at http://logs.glob.uno/?c=mozilla%23pdfjs&s=28+Apr+2017&e=28+Apr+2017#c60549). Such a check would disallow `null`, so we chose for this. Note that we already have this at the moment in https://github.com/mozilla/pdf.js/blob/master/make.js#L31.